dnf/module/module_base.py

```diff
@@ -497,6 +497,13 @@ def _get_info(self, module_specs):
                 lines["Repo"] = modulePackage.getRepoID()
                 lines["Summary"] = modulePackage.getSummary()
                 lines["Description"] = modulePackage.getDescription()
+                req_set = set()
+                for req in modulePackage.getModuleDependencies():
+                    for require_dict in req.getRequires():
+                        for mod_require, stream in require_dict.items():
+                            req_set.add("{}:[{}]".format(mod_require, ",".join(stream)))
+                if req_set:
+                    lines["Requires"] = "\n".join(sorted(req_set))
```

Comment:
It's the only "conditional" line in the module info output. Shouldn't be the `if req_set` line removed and `Requires` added to output even when empty?

Reply:
Hm, looks like consequent `self._create_simple_table(lines)` will strip any empty fields... So the previous comment is not relevant here.